### ISSUE TITLE
FR: SVG Native without click-event-handlers

### ISSUE THREAD

**Issue by lukemt** [zsviczian/obsidian-excalidraw-plugin#1265]
**Is your feature request related to a problem? Please describe.**
`SVG Image` does not render embeds.
`Native SVG` overwrites the click handlers.

**Describe the solution you'd like**
Maybe a fourth image type option would be cool: SVG Native without event handlers.
A rendering option that would render all the embeds, but not allow interacting with them. 

**Additional context**
This is just a tiny suggestion and I'm not even sure if I will stick to it, but I have already implemented this as a snippet:
```css
.markdown-source-view .excalidraw-svg svg {
  pointer-events: none;
}
```
So I personally don't need the feature (since I already have the snippet) but I share it because maybe others would like this

**Comment by lukemt** [comment]
I personally switched back to `SVG Image` for performance reasons.

**Comment by zsviczian** [comment]
yes, native SVG is a performance hit. That is why I did not enable it as default.
Thanks for sharing the snippet.
